core/src/main/java/org/apache/carbondata/core/datastore/page/statistics/PrimitivePageStatsCollector.java

```diff
@@ -140,6 +149,10 @@ private PrimitivePageStatsCollector(DataType dataType) {
       minDouble = Double.POSITIVE_INFINITY;
       maxDouble = Double.NEGATIVE_INFINITY;
       decimal = 0;
+    } else if (dataType == DataTypes.FLOAT) {
+      minFloat = Float.MIN_VALUE;
```

Comment:
It should be reverse, please check other datatypes. minFloat should be Float.MAX_VALUE

Reply:
done